README.md

```diff
@@ -240,6 +241,10 @@ Use `0` instead of `none` to specify that a style has no border.
     }
     ```
 
+### Variables
+
+Prefer dash-cased variable names (e.g. `$my-variable`) over camelCased or snake_cased variable names. It is acceptable to prefix variable names that are intended to be used only within the same file with an underscore (e.g. `$_my-variable`).
```

Comment:
i'm not a fan of the underscore prefixing here - using signals to indicate privacy where no privacy actually exists seems problematic.